ui/src/pageLayout/utils/navItemActivation.ts

```diff
@@ -1,8 +1,14 @@
+// Constants
+import {HOMEPAGE_PATHNAME} from 'src/pageLayout/constants/navItemActivation'
+
 export const getNavItemActivation = (
```

Comment:
should this be the default export as well? or do you envision other things coming out of this file?